### DISCUSSION TITLE
Match with different app store accounts

### DISCUSSION THREAD

**Discussion by mj-moor** [fastlane/fastlane#20905] [Q&A]
Hi,

I have a questions regarding Fastlane Match. I want to set this up for my project but I'm having difficulties wrapping my head around something. 

Situation:
We have two different app store accounts maintaining certificates and provisioning profiles for different build flavours. One app store account is for Debug builds and the other app store account is for our Prod builds. This is not something we can change.

My question; Can I maintain certificates and profiles for both app store accounts in one Git repo? My idea is to work with ENV variables to load the different app store users per need. Will it be possible two distinguish between App store users based on a Git Branch?

Thanks for the help!

**Comment by AlixH** [comment]
Did you get any solution for this use case ?
